src/platform/cannonlake/include/platform/lib/shim.h

```diff
@@ -148,6 +148,21 @@
 #define SHIM_CLKCTL_HMCS_DIV2	0
 #define SHIM_CLKCTL_HMCS_DIV4	BIT(0)
 
+/* Core clock PLL divisor */
+#define SHIM_CLKCTL_DPCS_MASK(x)	BIT(2)
+
+/* Prevent Audio PLL Shutdown */
+#define SHIM_CLKCTL_TCPAPLLS	BIT(7)
+
+/* 0--from PLL, 1--from oscillator */
+#define SHIM_CLKCTL_HDCS	BIT(4)
+
+/* Oscillator select */
+#define SHIM_CLKCTL_HDOCS	BIT(2)
+
+/* HP memory clock PLL divisor */
+#define SHIM_CLKCTL_HPMPCS	BIT(0)
+
 /** \brief Mask for requesting clock
```

Comment:
This looks to be new macros added. Care to add them in a separate patch as they are not matching the current commit description?

Reply:
@dbaluta ACK